src/js/tour.js

```diff
@@ -112,18 +112,12 @@ export class Tour extends Evented {
    * @param {String} event The event name to trigger
    */
   done(event) {
-    if (this.currentStep) {
-      this.currentStep.hide();
+    if (!isEmpty(this.steps)) {
+      this.steps.forEach((step) => step.destroy());
     }
 
     this.trigger(event);
 
-    if (Shepherd.activeTour) {
```

Comment:
Isn't this removed code here effectively doing the same thing as the added code above? I'm curious if there is a benefit to switching to `this.steps.forEach` over `Shepherd.activeTour.steps.forEach`?

Reply:
My answer to the second question is that my answer to the first question is "I don't know" 😄 

If it's not necessary to go through `Shepherd.activeTour.steps`, I think using `this.steps` is a lot more explicit about our intent.

Reply:
I think the idea is you would only want to destroy steps from the currently active tour. `this.steps` accomplishes roughly the same thing, but I think the main difference is if there is no active tour when `done` is called, there is no need to bother tearing anything down.

Reply:
🤔 But if `done` is called on the tour instance, shouldn't that instance only care about itself?

It seems like this would risk a) not tearing down the steps of the instance that a user calls `done` on, and b) accidentally tearing down steps of a separate instance.

Both of those seem like things we -- or users -- wouldn't want.